helm/README.MD

```diff
@@ -0,0 +1,272 @@
+# flyte
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.8.0](https://img.shields.io/badge/AppVersion-0.8.0-informational?style=flat-square)
+
+A Helm chart for Flyte
+
+### SANDBOX INSTALLATION:
+- [Install helm 3](https://helm.sh/docs/intro/install/) 
+- Fetch chart dependencies `helm dep up`
+- Install Flyte sandbox:
+
+```bash
+cd helm
+helm install -n flyte -f values-sandbox.yaml --create-namespace flyte .
+```
+
+Customize your installation by changing settings in `values-sandbox.yaml`.
+You can use the helm diff plugin to review any value changes you've made to your values:
+
+```bash
+helm plugin install https://github.com/databus23/helm-diff
+helm diff upgrade -f values-sandbox.yaml flyte .
+```
+
+Then apply your changes:
+```bash
+helm upgrade -f values-sandbox.yaml flyte .
+```
+
+#### Alternative: Generate raw kubernetes yaml with helm template
+- `helm template --name-template=flyte-sandbox . -n flyte -f values-sandbox.yaml > flyte_generated_sandbox.yaml`
+- Deploy the manifest `kubectl apply -f flyte_generated_sandbox.yaml`
+
+
+- When all pods are running - run end2end tests: `kubectl apply -f ../end2end/tests/endtoend.yaml`
+- Get flyte host `minikube service contour -n heptio-contour --url`. And then visit `http://<HOST>/console`
+
+
+### CONFIGURATION NOTES:
+- The docker images, their tags and other default parameters are configured in `values.yaml` file.
+- Each Flyte installation type should have separate `values-*.yaml` file: for sandbox, EKS and etc. The configuration in `values.yaml` and the choosen config `values-*.yaml` are merged when generating the deployment manifest.
+- The configuration in `values-sandbox.yaml` is ready for installation in minikube. But `values-eks.yaml` should be edited before installation: s3 bucket, RDS hosts, iam roles, secrets and etc need to be modified.
+
+## Values
+
+| Key | Type | Default | Description |
```

Comment:
ohh man this is crazy, maintaining this will be a nightmare right? We need to auto-generate this

Reply:
Completely agree. Looking at 6517fbbda32cdc6d9e922226342e0ce9d7e912a2 it seems it's already autogenerated

@rstanevich did you use https://github.com/norwoodj/helm-docs?

Reply:
@sbrunk
Yes, this README is autogenerated using the helm plugin https://github.com/norwoodj/helm-docs and template file https://github.com/flyteorg/flyte/blob/125710b4c2b23a0a39e794a0761c4337872467ce/helm/README.md.gotmpl